src/sentry/notifications/notifications/active_release.py

```diff
@@ -0,0 +1,240 @@
+                release_version = last_release.version if last_release else None
+                event = shared_context.get("event", None)
+                for participant in participants:
+                    self.record_active_release_notification_sent(
+                        participant, event, provider, release_version
+                    )
+                if (
+                    features.has("organizations:active-release-monitor-alpha", self.organization)
+                    and last_release
+                ):
+                    ReleaseActivity.objects.create(
+                        type=ReleaseActivityType.ISSUE.value,
+                        data={
+                            "provider": EXTERNAL_PROVIDERS[provider],
+                            "group_id": self.group.id,
+                        },
+                        release=last_release,
+                    )
+            notify(provider, self, participants, shared_context)
+
+    def get_context(self) -> MutableMapping[str, Any]:
+        environment = self.event.get_tag("environment")
+        enhanced_privacy = self.organization.flags.enhanced_privacy
+        group = self.group
+        context = {
+            "project_label": self.project.get_full_name(),
+            "group": group,
+            "users_seen": self.group.count_users_seen(),
+            "event": self.event,
+            "link": get_group_settings_link(
+                group, environment, rule_details=None, referrer="alert_email_release"
+            ),
+            "has_integrations": has_integrations(self.organization, self.project),
+            "enhanced_privacy": enhanced_privacy,
+            "last_release": self.last_release,
+            "last_release_link": self.release_url(self.last_release),
+            "last_release_slack_link": self.slack_release_url(self.last_release),
+            "environment": environment,
+            "slack_link": get_integration_link(self.organization, "slack"),
+            "has_alert_integration": has_alert_integration(self.project),
+            "regression": self.event_state.is_regression
+            if self.event_state and self.event_state.is_regression
+            else False,
+        }
+        # if the organization has enabled enhanced privacy controls we don't send
+        # data which may show PII or source code
+        if not enhanced_privacy:
+            contexts = (
+                self.event.data["contexts"].items() if "contexts" in self.event.data else None
+            )
+            event_user = self.event.data["event_user"] if "event_user" in self.event.data else None
+            context.update(
+                {
+                    "tags": self.event.tags,
+                    "interfaces": get_interface_list(self.event),
+                    "contexts": contexts,
+                    "event_user": event_user,
+                }
+            )
+
+        return context
+
+    def record_active_release_notification_sent(
```

Comment:
@barkbarkimashark Do you need to make this custom function for recording notifications? We can't just use `analytics_event` and `get_custom_analytics_params` to populate this?

Reply:
Yeah, this will get addressed in my other PR after I rebase that with this.